backend/LearningCalendar/Epicenter.Service/Employee/EmployeeService.cs

```diff
@@ -24,5 +37,61 @@ public async Task<List<EmployeeDto>> GetEmployees()
                 {
                 }).ToList();
         }
+
+        public async Task<EmployeeDto> CreateAsync(string identityId, string managerEmail)
+        {
+            if (managerEmail == null)
+            {
+                var topLevelManager = await _employeeRepository.GetTopLevelManagerAsync();
+                if (topLevelManager != null)
+                {
+                    throw new ApplicationException("Top level manager already exists");
+                }
+            }
+
+            return await CreateEmployeeWithoutChecksAsync(identityId, managerEmail);
+        }
+
+        private async Task<EmployeeDto> CreateEmployeeWithoutChecksAsync(string identityId, string managerEmail)
+        {
+            var identityTask = _identityRepository.QuerySingleAsync(identity => identity.Id == identityId);
+            
+            var globalLimitTask = _limitRepository.GetGlobalAsync();
+            var manager = await _employeeRepository.GetByEmailAsync(managerEmail);
+
+            Team team = null;
+            if (manager != null)
+            {
+                var teamDto = await _teamService.GetOrCreateForManagerAsync(manager.Id);
+                var teamTask = _teamRepository.GetByManagerIdAsync(teamDto.ManagerId);
+                team = await teamTask;
+            }
+
+            var employee = new Domain.Entity.LearningCalendar.Employee
+            {
+                Id = Guid.NewGuid(),
+                Identity = await identityTask,
+                Team = team,
+            };
+
+            await _employeeRepository.CreateAsync(employee);
+
+
+            var limit = await globalLimitTask;
+            if (limit == null)
+            {
+                limit = new Limit
+                {
+                    Id = Guid.NewGuid(),
+                    Creator = employee,
+                };
+                await _limitRepository.CreateAsync(limit);
+            }
+            employee.Limit = limit;
+
+            await _employeeRepository.UpdateAsync(employee);
+            
+            return new EmployeeDto();

```

Comment:
I see that in `TeamService` you `set` properties of `TeamDto` but don't do it here. Why?

Reply:
@zigsas because for now the `EmployeeDto` has two props `FirstName`, `LastName`, which are not used anywhere, and we don't even save that in the db, so until something needs something in the employeeDto, I'll just leave it as is.